sotodlib/preprocess/processes.py

```diff
@@ -47,10 +50,16 @@ class DetBiasFlags(_Preprocess):
     name = "det_bias_flags"
 
     def calc_and_save(self, aman, proc_aman):
-        msk = tod_ops.flags.get_det_bias_flags(aman, merge=False,
-                                               **self.calc_cfgs)
+        if self.plot_cfgs is None:
```

Comment:
I think it's better to modify `flags.det_bias_flags` to always return an aman and if `full_output = True` have it wrap all of the submasks but also the main mask (called det_bias_flags). Here in the process let's just have it always run with full_output=True (not just when plot_cfgs is present) these two things should get rid of the two different mans you have here (dbc_aman and msk_aman)